src/main/java/net/pms/configuration/FormatConfiguration.java

```diff
@@ -115,6 +119,7 @@
 	public static final String RAW = "raw";
 	public static final String REALAUDIO_14_4 = "ra14.4";
 	public static final String REALAUDIO_28_8 = "ra28.8";
+	public static final String RGB = "rgb";
```

Comment:
Don't you see how that will make the code much easier to maintain in the future? Now, if you have never seen this code before, or have forgotten why you did this, and is to refactor something that touches this - how are you to understand what ```RGB``` or ```YUV``` means as a ```FormatConfiguration```? It doesn't make sense since those are different color encodings and not any particular codec or format. With the description, you have a chance to understand what it means, it means a stream of multiple uncompressed images where the bits are coded as ```RGB``` or ```YUV```. That makes it much easier to make the correct decision when refactoring, and that is what I'm after. I look at it as preventing future bugs.